packages/node_modules/@cerebral/http/README.md

```diff
@@ -150,7 +150,7 @@ const errorCatched = [
   displayError
 ]
 
-Controller({
+Module({
   catch: new Map([
```

Comment:
`new Map()` should also be removed and "signal/global catch handlers" above should be "module catch handlers".